src/restserver.h

```diff
@@ -52,6 +53,9 @@ typedef struct
 
     // rest-subsciptions
     rest_list_t *observeList;
+
+    char *certificate;
```

Comment:
Just save the reference to settings. Will be able to access information in future more easily.